doc/contributor/howto-adding-generated-libraries.md

```diff
@@ -1,11 +1,16 @@
-# Adding new Libraries to `google-cloud-cpp`
+# HOWTO: adding generated libraries
```

Comment:
s/adding/Adding/

Reply:
Done. Also made the file name and title more consistent with the other docs in this directory.